src/test/java/com/exasol/artifactreferencechecker/ArtifactReferenceCheckerMojoIT.java

```diff
@@ -1,97 +1,90 @@
 package com.exasol.artifactreferencechecker;
 
-import static org.hamcrest.CoreMatchers.*;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.not;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 import java.io.File;
 import java.io.IOException;
+import java.nio.file.Files;
 import java.nio.file.Path;
 
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.Test;
-import org.testcontainers.containers.BindMode;
-import org.testcontainers.containers.Container.ExecResult;
-import org.testcontainers.containers.GenericContainer;
-import org.testcontainers.junit.jupiter.Container;
-import org.testcontainers.junit.jupiter.Testcontainers;
+import org.apache.commons.io.FileUtils;
+import org.apache.maven.it.VerificationException;
+import org.apache.maven.it.Verifier;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.io.TempDir;
 
-@Testcontainers
-public class ArtifactReferenceCheckerMojoIT {
-    private static final File PLUGIN = Path.of("target", "artifact-reference-checker-maven-plugin-0.3.1.jar").toFile();
-    private static final File PLUGIN_POM = Path.of("pom.xml").toFile();
-    private static final File TEST_PROJECT = Path.of("src", "test", "resources", "unit", "test_project").toFile();
+import com.exasol.mavenpluginintegrationtesting.MavenIntegrationTestEnvironment;
+import com.exasol.mavenprojectversiongetter.ProjectVersionGetter;
```

Comment:
I think this should be `MavenProjectVersionGetter`, builds seems to fail because of it.